pkg/i18n/translations/ja.json

```diff
@@ -10,7 +10,6 @@
   "MainTitle": "メイン",
   "StagingTitle": "メインパネル (Staging)",
   "MergingTitle": "メインパネル (Merging)",
-  "MergeConfirmTitle": "マージ",
```

Comment:
What's going on here, did you edit these manually? I don't think we should do that, it's error-prone and unnecessary. The source of truth for these files should be crowdin (once we have it), until then we should simply leave these untouched.

Reply:
Indeed: I edited them manually. If crowdin is going to be the new source of truth for these translation files, I'm happy to leave them alone.

I've just set up a crowdin account and sent you an invite. I've also applied for the open-source licence so that it's free